### ISSUE TITLE
Failure to achieve the expected accuracy rate

### ISSUE THREAD

**Issue by tuningManBin** [Brother-Lee/WRN-28-10-CIFAR10#2]
Hello, thanks for sharing the code.
I ran 200 epochs, but the best accuracy was only about 90%.
Is there any strategy for adjusting the learning rate during training?
